src/command_modules/azure-cli-vsts/azure/cli/command_modules/vsts/custom.py

```diff
@@ -186,11 +186,9 @@ def list_releases(name, resource_group_name):
 
     headers = {'Content-type': 'application/json', 'Authorization': o_auth_token}
     req = requests.get(get_releases_url, headers=headers, timeout=600)
-    json_request = req.json()
     if req.status_code != 200:
-        print('Error: ' + str(req.status_code) + '. Could not list releases: ' +
-              json_request["error"]["code"])
-        raise CLIError(req.text)
+        raise CLIError('Error: ' + str(req.status_code) + '. Could not list releases: ' + req.text)
```

Comment:
Please remove the string 'Error: '.
When raising a CLIError, you don't need to add this prefix as the CLI will handle outputting the error.
If it's being logged to file, 'ERROR' will be added as a prefix.
If it's logged to a console that supports color, it'll appear in red.